### DISCUSSION TITLE
A Unix/Epoch date readable by ordinary mortals?

### DISCUSSION THREAD

**Discussion by leodium** [cwmonkey/nms-expeditions#11] [Ideas]
On https://cwmonkey.github.io/nms-expeditions/ , the date **-Expedition End Date-** is entered in unix/epoch format ... Wouldn't it be more legible to put it in dd:mm:yy format?

**Comment by cwmonkey** [comment]
@leodium Does that actually work?

**Comment by leodium** [comment]
Yes ... But in epoch format 🙈

**Reply by leodium** [reply]
![Screenshot_20240425_055237.jpg](https://github.com/cwmonkey/nms-expeditions/assets/3294203/5e11c788-1f64-4039-bb1d-8df6b7d08462)

**Reply by cwmonkey** [reply]
Oh I see, you're saying change how it displays just on the site. Yeah I'll get around to that some day.
